servo-gecko-try/init.sls

```diff
@@ -0,0 +1,91 @@
+{% from tpldir ~ '/map.jinja' import servo-gecko-try %}
+
+user-acct:
+  user.present:
```

Comment:
Move this under the `servo-gecko-try` ID. The `name` parameter can then be omitted.
TODO for me: I need to double check if this will also create the group automatically.